WordPress/gutenberg
# [Mobile] Create cover block with only colour (from a default palette) without an image

**Fixes** https://github.com/WordPress/gutenberg/issues/23831 (**parent** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2275)

## Related PRs:
* Gutenberg-Mobile https://github.com/wordpress-mobile/gutenberg-mobile/pull/2488
* Clear media button in block settings https://github.com/WordPress/gutenberg/pull/23794
* Create cover block with custom colour https://github.com/WordPress/gutenberg/pull/23870
* Media Button when only colour is selected https://github.com/WordPress/gutenberg/pull/23878

## Description
A palette with four colours ([black, white and a couple of theme defaults](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2275#issuecomment-635095905)) has been added bellow the Add Media Button. When the user taps on a colour indicator the block is created with the selected colour as background.
The latest design can be found [here](https://github.com/WordPress/gutenberg/pull/23870#issuecomment-656923629).

## How has this been tested?
* Add a cover block
* Press the one colour from the palette
* Verify that a new cover block with only colour is created

## Screenshots

|Empty cover block|Android|iOS|
|---|---|---|
|![Simulator Screen Shot - iPhone 11 - 2020-07-14 at 17 34 57](https://user-images.githubusercontent.com/304044/87438579-64af6900-c5f8-11ea-9c41-311eff429875.png)|![android](https://user-images.githubusercontent.com/304044/87438044-bacfdc80-c5f7-11ea-99d9-bbc981030dd9.gif)|![ios](https://user-images.githubusercontent.com/304044/87438034-b6a3bf00-c5f7-11ea-93fe-e9a3b3f3ace4.gif)|

## Types of changes
Enhancement

## Checklist:
